src/plugins/camera_controller_manager/CameraControllerManager.hh

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef IGNITION_GUI_PLUGINS_CAMERACONTROLLERMANAGER_HH_
+#define IGNITION_GUI_PLUGINS_CAMERACONTROLLERMANAGER_HH_
+
+#include <memory>
+
+#include "ignition/gui/Plugin.hh"
+
+namespace ignition
+{
+namespace gui
+{
+namespace plugins
+{
+  class CameraControllerManagerPrivate;
+
+  class CameraControllerManager : public Plugin
```

Comment:
By its name, this plugin could be confused with `ViewAngle`. How about `CameraFollow`, or `CameraTracking`?